### DISCUSSION TITLE
What tool was used to generate this relationship diagram?

### DISCUSSION THREAD

**Discussion by reaink** [prisma/prisma#11756] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/45023042/153560210-bf6d06b9-6faa-4b20-968c-537d0aed14bf.png)
What tool was used to generate this relationship diagram?

**Comment by nurul3101** [comment, marked as answer]
Hey @nshusr 👋🏻 ,

Most of the diagrams and illustrations that you see in docs are created by our designer team in [Figma](https://www.figma.com/).

However if you want to generate ER Diagrams from your Prisma Schema there are various community solutions that could be useful to you.

Please feel free to checkout any of these:

https://github.com/notiz-dev/prisma-dbml-generator
https://github.com/emyann/prisma-uml
https://prisma-erd.simonknott.de/

Please let me know if you need help with something else.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
